src/coreclr/jit/emitarm64.h

```diff
@@ -316,14 +316,59 @@ static code_t insEncodeReg_Vm(regNumber reg);
 // Returns an encoding for the specified register used in the 'Va' position
 static code_t insEncodeReg_Va(regNumber reg);
 
-// Returns an encoding for the specified register used in the 'Pd' position
-static code_t insEncodeReg_Pd(regNumber reg);
+// Return an encoding for the specified register used in '4' thru '0' position.
```

Comment:
If there's a comment here, it seems like it should match the definition comment, e.g.:
```
Return an encoding for the specified 'V' register used in '4' thru '0' position.
```

(same below)

Reply:
Done.